Assignment5/AlvaroAlonso-0-b190300/kgtk_notebook.py

```diff
@@ -0,0 +1,274 @@
+                  output_path=output_path,
+                  project_name=project_name)
+
+"""The KGTK setup command defines environment variables for all the files so that you can reuse the Jupyter notebook when you install it on your local machine."""
+
+ck.print_env_variables()
+
+# Commented out IPython magic to ensure Python compatibility.
+# %%time
+ck.load_files_into_cache()
+
+"""# About this assignment.
+This assignment is based on https://github.com/usc-isi-i2/kgtk-notebooks/tree/main/tutorial. If you have any questions or doubts, it is encouraged to look how the tutorial performs the different operations.
+
+Additional information can be found in https://kgtk.readthedocs.io/
+
+## Simple graph statistics
+
+Let's calculate first some statistics about the KG. Count the number of instances:
+"""
+
+# Commented out IPython magic to ensure Python compatibility.
+# %%time
+kgtk("""
+    query -i all
+      --match '()-[]->()'
+      --return 'count()'
+""")
+
+"""Now, count the number of distinct properties: 
+
+"""
+
+# Commented out IPython magic to ensure Python compatibility.
+# %%time
+kgtk("""
+    query -i all
+      --match '()-[p]->()'
+      --return 'count(distinct p.label)'
+""")
+
+"""Now, let's count the frequency of those properties. That is, how many instances we can find with each property"""
+
+# Commented out IPython magic to ensure Python compatibility.
+# %%time
+kgtk("""
+    query -i all
+      --match '()-[p]->()'
+      --return 'p.label, count(p)'
+""")
+
+"""## Simple queries
+Some of these queries are simple and will run in the Wikidata endpoint. 
+Try both of them using SPARQL and Kypher
+"""
+
+# Which actors has Schwarzenegger worked with throughout his career? (Print also the movie)
+
+# (in SPARQL)
+"""
+SELECT ?acName (GROUP_CONCAT(?mName; SEPARATOR=", ") as ?movie) WHERE{
+  ?movie wdt:P31 wd:Q11424 .
+  ?movie wdt:P161 wd:Q2685 .
+  ?movie rdfs:label ?fName FILTER (lang(?mName) = "es") .
+  ?movie wdt:P161 ?ac FILTER (?actor != wd:Q2685) .
+  ?actor rdfs:lavel ?acName FILTER (lang(?acName) = "es") . 
+}GROUP BY ?actor ?acName
+"""
+
+# In Kypher:
+kgtk("""
+    query -i all
+      --match '(:Q11424)<-[:P31]-(m)-[:P161]->(:Q2685), (mName)<-[:label]-(m)-[:P161]->(actor)-[:label]->(acName)'
+      --where 'actor != "Q2685"'
+      --return 'acName as Name, mName as Movie'
+""")
+
+# How many awards does Schwarzenegger have?
+
+# SPARQL:
+"""
+SELECT (count(?award) as ?nAwards) WHERE{
+  wd:Q2685 wdt:P166 ?award
+}
+"""
+
+# Kypher:
+kgtk("""
+    query -i all
+    --match '(:Q2685)-[:P166]->()'
+    --return 'count()'
+""")
+
+# Retrieve at least two members of Schwarzenegger's political party. Make sure only persons are returned
+# SPARQL:
+"""
+SELECT ?partyName WHERE{
+  wd:Q2685 wdt:P102 ?poliParty .
+  ?member wdt:P102 ?poliParty FILTER (?member != wd:Q2685) FILTER NOT EXISTS {?member wdt:P570 []} .
+  ?member rdfs:label ?partyName FILTER (lang(?partyName) = "es") .
```

Comment:
here you are not ensuring the members are persons

Reply:
Just pushed the changes requested

Reply:
Yes, I forgot to save the last change before the commit. Now should be correct